dataconfig/parsers.py

```diff
@@ -251,6 +278,11 @@ def _validator(key: str, value: Dict) -> Dict[str, classmethod]:
     return make_validator(func, key, opts=opts, **params)
 
 
+def _transformation(key: str, value: Dict) -> classmethod:
+    factory = getattr(NS.transforms, value[_type_spec[9]])
+    return factory()
```

Comment:
Simplify the transformation implementation to a regular function, and return `classmethod(mytransform)` here.